### PR TITLE
gsutil failure bug

### PR DESCRIPTION
**Pull request by allieychen**
Updates the base image to cloud-genomics-pipelines/io to use the wrapped gsutil with additional retry logic.

Note: no public image is published yet.
Tested: manually builded the image and tested on the image.

### CODE REVIEW

File: batch/vep/run_script_with_watchdog.sh [hunk 1]
```diff
@@ -23,18 +23,13 @@
 set -euo pipefail
 
 #################################################
-# Returns the generation number of a GCS file. In case of failure, it sleeps and
-# retries for at most 3 times.
+# Returns the generation number of a GCS file. Use the wrapped gsutil in
```
Comment (kemp-google) [inline batch/vep/run_script_with_watchdog.sh]: I would put this comment at the top where you define the base image.

`The pipelines-io container provides a wrapper around gsutil with additional retry logic.`

File: batch/vep/run_script_with_watchdog.sh [hunk 2]
```diff
@@ -23,18 +23,13 @@
...
 #   $1: The GCS file.
 #################################################
 function get_last_update_time {
-  num_retries=0
-  until last_update_time_in_microseconds="$(gsutil stat $1 | awk '$1 == "Generation:" {print $2}')" || [ ${num_retries} -eq 4 ]; do
-    ((num_retries++))
-    time_to_sleep_sec="$((num_retries*1))"
-    sleep "${time_to_sleep_sec}"
-  done
+  last_update_time_in_microseconds="$(gsutil stat $1 | awk '$1 == "Generation:" {print $2}')"
```
Comment (kemp-google) [inline batch/vep/run_script_with_watchdog.sh]: Can you replace this with just the invocation?

function get_last_update_time {
  gsutil stat $1 | awk '$1 == Generation:" {print $2}'
}

File: batch/vep/run_script_with_watchdog.sh (hunk 1 above)
Comment (allieychen) [reply batch/vep/run_script_with_watchdog.sh]: Done.

File: batch/vep/run_script_with_watchdog.sh (hunk 2 above)
Comment (allieychen) [reply batch/vep/run_script_with_watchdog.sh]: Done.

**Comment by allieychen** [comment]
Thank you Aaron! PTAL :)

**Review by kemp-google — APPROVED** [review]
Please make sure you update the commit message with the content from your PR message before rebasing, but otherwise LGTM

**Pull request merged**